configuration/rules-dsl.md

```diff
@@ -204,7 +204,37 @@ You can find it from PaperUI or from Karaf remote console.
 For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
-> The parser will not recognize the whole text correctly without quotes.
+
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+Some add-ons provide trigger channels. 
+Your rules can take actions based upon trigger events generated by these trigger channels.
+You can decide whether you want to catch only a specific or any trigger the channel provides.
+Here is the syntax for these cases (parts in square brackets are optional):
+
+```java
+Channel "<triggerChannel>" triggered [<triggerEvent>]
+```
+
+`triggerChannel` is the identifier for a specific channel.
+
+When a binding provides such channels, you can find the needed information in the corresponding binding documentation.
+There is no generic list of possible values for `triggerEvent`,
+The `triggerEvent`(s) available depend upon the specific implementation details of the binding.
+
+Example:
+
+```java
+rule "Start wake up light on sunrise"
+when
+    Channel "astro:sun:home:rise#event" triggered START
+then
+    ...
+end
+```
+
+> Note: You need to use quotes around `triggerChannel` if it contains special characters such as ':'.
```

Comment:
Wondering if this one could be moved to the trigger channel description above?

Reply:
That would make sense - I vote yes.